src/sql/workbench/services/notebook/browser/models/notebookModel.ts

```diff
@@ -264,6 +265,10 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		return this._kernelsChangedEmitter.event;
 	}
 
+	public get kernelsAdded(): Event<nb.IKernel> {
```

Comment:
Please add a doc comment explaining when this event is fired (and it'd be great to do that for the other events as well since clearly there's some confusion with the naming)